src/karmabot/slack.py

```diff
@@ -30,22 +31,24 @@
 AUTOMATED_COMMANDS = {"welcome": welcome_user}  # not manual
 ADMIN_BOT_COMMANDS = {"top_karma": top_karma}
 PUBLIC_BOT_COMMANDS = {
-    "age": pybites_age,
     "add": add_command,
+    "age": pybites_age,
     "help": create_commands_table,
+    "joke": joke,
```

Comment:
was this dict not complete?

Reply:
We use `isort` so maybe "always sort by import order"?

Reply:
I see, i agree :) will undo this change

Reply:
or better take the suggestion and reorder it according to the imports good point